src/XliffTasks/Tasks/XlfTask.cs

```diff
@@ -20,7 +20,7 @@ public sealed override bool Execute()
             }
             catch (BuildErrorException ex)
             {
-                Log.LogErrorFromException(ex, showStackTrace: false);
+                Log.LogErrorFromException(ex, showStackTrace: false, showDetail: true, file: (string)ex.Data[BuildErrorException.RelatedFile]);
```

Comment:
This gives us output like so:
```
"c:\XliffTasksTest\XliffTasksTest.sln" (UpdateXlf target) (1) ->
"c:\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj" (UpdateXlf target) (2) ->
(_UpdateXlf target) -> 
  xlf\MyResources.cs.xlf : error : BuildErrorException: Unable to load file. [c:\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj]
xlf\MyResources.cs.xlf : error : XmlException: The 'trans-unit' start tag on line 10 position 8 does not match the end tag of 'body'. Line 11, position 7. [c:\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj]
xlf\MyResources.cs.xlf : error :  [c:\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj]
```

Reply:
My intent with BuildErrorException was that it is a regular build error implemented as an exception. So the fact that the user sees "BuildErrorException" here goes against that.

Reply:
So you're suggesting I remove `showDetail: true`?

Reply:
Note that without `showDetail: true` we would get the following output:

```
"c:\Users\tomescht\source\repos\XliffTasksTest\XliffTasksTest.sln" (rebuild target) (1) ->
"c:\Users\tomescht\source\repos\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj" (Rebuild target) (2) ->
(_UpdateXlf target) ->
  xlf\MyResources.cs.xlf : error : Unable to load file. [c:\Users\tomescht\source\repos\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj]
```

You lose the detail from the inner exception that helps you find the error.

Reply:
We control the message on all BuildErrorExceptions. It's our type. Think of it as `LogError(msg); StopBuild();` If you want to put more than "Unable to load file, " then format the xml exception into the builderrorexception message.

Reply:
That leaves us with:

```
"c:\Users\tomescht\source\repos\XliffTasksTest\XliffTasksTest.sln" (rebuild target) (1) ->
"c:\Users\tomescht\source\repos\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj" (Rebuild target) (2) ->
(_UpdateXlf target) ->
  xlf\MyResources.cs.xlf : error : Unable to load file: The 'trans-unit' start tag on line 10 position 8 does not match the end tag of 'body'. Line 11, position 7. [c:\Users\tomescht\source\repos\XliffTasksTest\XliffTasksTest\XliffTasksTest.csproj]
```